internal/provider/resource_string.go

```diff
@@ -442,6 +450,13 @@ func stringSchemaV3() schema.Schema {
 					boolplanmodifiers.NumberNumericAttributePlanModifier(),
 					boolplanmodifier.RequiresReplace(),
 				},
+				Validators: []validator.Bool{
+					validators.AtLeastOneOfTrue(
```

Comment:
Nota Bene: Definitely don't need to change this, but another consideration here might be to introduce this multiple attribute validation at the resource level as a ConfigValidator and specify all 4 attributes in there. Maybe its just me but I tend to get a little confused reading attribute-based validators (remembering "oh yeah, it includes this attribute too") and whether that attribute-based validation should be added to all the relevant attributes (confusingly, no, since it will generate multiple similar diagnostics). 😅

Reply:
Happy to refactor along these lines, but if you're ok with the current implementation could you approve the PR?